### DISCUSSION TITLE
import { BottomSheet } results in 47.9kb gzipped import

### DISCUSSION THREAD

**Discussion by bscaspar** [stipsan/react-spring-bottom-sheet#167] [Q&A]
I am working on reducing the bundle size of my app, but `react-spring-bottom-sheet` seems to have a significantly larger import size than any other library I use. I noticed the readme says 'gzip size: 6.31 kB' but both the import cost analyzer and bundle analyzer show that this import is significantly larger. I am using `v3.4.0`

Am I using this wrong? Why would the chunk be so large?

use: 
```ts
import React from 'react';

import { BottomSheet } from 'react-spring-bottom-sheet';
import 'react-spring-bottom-sheet/dist/style.css';

import styles from './BottomSheetModal.module.scss';

type BottomSheetModalProps = {
  visible: boolean;
  children: React.ReactNode;
  onClose: () => void;
};

const BottomSheetModal = ({
  visible,
  children,
  onClose,
}: BottomSheetModalProps): JSX.Element => (
  <BottomSheet
    className={styles.bottomSheetModal}
    open={visible}
    onDismiss={onClose}
  >
    <div className={styles.bottomSheetModalContent}>{children}</div>
  </BottomSheet>
);

export default BottomSheetModal;
```

import cost analyzer output:
<img width="546" alt="Screen Shot 2021-09-06 at 4 32 54 PM" src="https://user-images.githubusercontent.com/6042704/132263700-adb7a125-e228-494a-b648-e650e4ba45bf.png">

bundle-analyzer output:
![Screen Shot 2021-09-06 at 4 33 20 PM](https://user-images.githubusercontent.com/6042704/132263715-32a3087c-e896-4f42-ad5e-4da5953ce525.png)

**Comment by rossthedevigner** [comment]
I'm seeing the same thing. @bscaspar Have you found any clues?

**Reply by bscaspar** [reply]
Nothing yet, although I admittedly haven't focused on this much. Bundlephobia seems to disagree with the unpkg report on the readme: https://bundlephobia.com/package/react-spring-bottom-sheet@3.4.0

**Comment by pepew-le-boss** [comment]
Up
